sources/zarr/large_image_source_zarr/__init__.py

```diff
@@ -717,7 +717,13 @@ def _writeInternalMetadata(self):
                         'inverted': False,
                         'label': f'Band {c + 1}',
                     }
-                    channel_data = base_array[..., c]
+                    slicing = tuple(
+                        slice(None)
+                        if k != ('C' if 'C' in self._axes else 'S')
```

Comment:
On line 677, self._axes is lowercase; here we are checking upper case.  I think this should be lowercase.

Reply:
Good point. Changed in [5e33da7](https://github.com/girder/large_image/pull/1557/commits/5e33da7ab9ab71ceb3e502208809f5e18744a9b6).